docs/plugins/cloud-aws.asciidoc

```diff
@@ -1,468 +1,8 @@
 [[cloud-aws]]
 === AWS Cloud Plugin
 
-The Amazon Web Service (AWS) Cloud plugin uses the
-https://github.com/aws/aws-sdk-java[AWS API] for unicast discovery, and adds
-support for using S3 as a repository for
-{ref}/modules-snapshots.html[Snapshot/Restore].
+The AWS Cloud Plugin has been split in two plugins:
```

Comment:
@clintongormley If I create a new `redirects.asciidoc` file within plugins documentation, will that work?
Or do I need to define something else?